app/assets/templates/static_pages/index.html

```diff
@@ -0,0 +1,12 @@
+<div class="container">
+  <div class="row">
```

Comment:
why do you need the `{{ message }}`? You commented it out in https://github.com/WDI-HK-8/mecha-assist/blob/loginsignin/app/assets/javascripts/angular/controller/articles/static_pages.js.erb#L2